src/inc/internal/AppxBundleWriter.hpp

```diff
@@ -0,0 +1,90 @@
+//
+//  Copyright (C) 2019 Microsoft.  All rights reserved.
+//  See LICENSE file in the project root for full license information.
+// 
+#pragma once
+
+#include "AppxPackaging.hpp"
+#include "ComHelper.hpp"
+#include "DirectoryObject.hpp"
+#include "AppxBlockMapWriter.hpp"
+#include "ContentTypeWriter.hpp"
+#include "ZipObjectWriter.hpp"
+#include "BundleWriterHelper.hpp"
+#include "BundleManifestWriter.hpp"
+#include "AppxPackageInfo.hpp"
+
+#include <map>
+#include <memory>
+#include <future>
+
+// internal interface
+// {32e89da5-7cbb-4443-8cf0-b84eedb51d0a}

```

Comment:
>32e89da5 [](start = 4, length = 8)

should match the generated guid you specify in MSIX_INTERFACE below

Reply:
Generated new guid and matched it with the one defined above.